src/main/java/gregtech/integration/jei/utils/JEIHooks.java

```diff
@@ -0,0 +1,27 @@
+package gregtech.integration.jei.utils;
```

Comment:
This code is still needed, as it is the code that is inserted into JEI via the coremod. Though, there may be a better place in the codebase for this file. I followed the pattern of our previous coremods, where the hook class/methods were in the same directory as other similar code, for example the Armor hooks being with other Armor rendering rather than with asm code.

Reply:
Fair enough let's leave it like this